infra/src/main/kotlin/fi/fta/geoviite/infra/tracklayout/TrackLayout.kt

```diff
@@ -334,7 +334,7 @@ data class TrackLayoutSwitchJointConnection(
 
 data class DraftableChangeInfo(
     val created: Instant,
-    val changed: Instant,
+    val changed: Instant?,
```

Comment:
Niin siis miksikäs tästä tuli null? Eikös muutosaika ole luonnollisesti = luontiaika jos ei ole muuta muutosta? 

Jos draftia/officialia ei ole niin koko changeinfon pitäisi ehkä olla null eikä vain muutosajan?  Toisaalta tuo created-aika on tainnut toimia niin että se on officialillekin version 1 aikaleima eikä ensimmäisen officialin aikaleima, joten ehkä koko olio ei voi olla null... Mitenkähän tämän pitäisi mennä.

En enää muista miten tuosta puhuttiin mutta näyttää vähän hassulta. Voisi käväistä vaikka dailyssä läpi.

Reply:
Ehkä tämän voisi tosiaan käydä dailyssä vielä läpi, sillä en nyt näin kolmannella mietinnällä ole itsekään täysin varma mikä tässä olisi haluttua ja mikä ei. Se syy miksi tämän tähän muotoon muutin oli, että draft-onlylle ei ole vielä olemassa virallista muutosaikaa, ja ajattelin että näissä tilanteissa olisi kivempi palauttaa null kuin heittää, koska tuo virhe kuitenkin palautuu frontille saakka. Mutta pitäisi varmaan keksiä tähän joku periaatepäätös, sekin lienee sinällään ihan validi vaihtoehto että palautetaan noissa tilanteissa draftin changeTime.